docs/walkthrough.md

```diff
@@ -514,6 +518,34 @@ kubectl delete ns test-ns catalog ups-broker
 
 ## Troubleshooting
 
+### Non-default image installation
+
+The service catalog installation will by default use images of the controller
+and apiserver built at the latest tagged version. Should you be using a
+different state of the repo (for instance, the latest commit in the `master`
+branch), there may be conflicts with these images and the latest chart and
+walkthrough files. During installation, you may specify the images of the
+controller and apiserver to use. To do so, set your `REGISTRY` and `VERSION`
+environment variables to the appropriate container registry and image tag
+respectively, and run the following:
+
+```console
+helm install charts/catalog --name catalog --namespace catalog --set apiserver.image=${REGISTRY}apiserver:${VERSION},controllerManager.image=${REGISTRY}controller-manager:${VERSION}
+```
+
+The `quay.io/kubernetes-service-catalog/` container registry stores images built
+from recent `master` commits. To use these images, set `REGISTRY` to point to
+this registry and `VERSION` to the abbreviated SHA of that commit via the
+following:
+
+```console
+REGISTRY=quay.io/kubernetes-service-catalog/
+VERSION=$(git describe --always --abbrev=7)
+```
+
+Note that there is a lag between when a commit is made and the images are built,
+so the latest commit to `master` may not yet be available.
```

Comment:
Since there is a delay, it may be worth pointing out the devguide section for building images locally and adjusting the imagePullPolicy. Though admittedly this raises the bar a bit for somebody just getting started.

Reply:
I'll add a quick mention to check out the build documentation in the devguide.